lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -39,6 +28,7 @@
 }
 
 
+@ddt.ddt
```

Comment:
I cleaned up these tests a bit as I was evaluating whether to keep them here or in `test_tasks.py`. I noticed that we were basically repeating the same tests twice for each of 'set' and 'reset', so I ddt'd those together.